fetch/api/resources/basic.html

```diff
@@ -0,0 +1 @@
+<!DOCTYPE html>
```

Comment:
That means I'd have to install a service worker at that scope. Is that likely to interfere with other tests?